### PR TITLE
Fix rescan regions during storage testing. Was returning first of lis…

### PR DESCRIPTION
**Pull request by sipickles**
…t of statuses, which is order undefined. Now we always test against europe.

### CODE REVIEW

**Comment by sipickles** [comment]
Still failing on Jenkins, investigating

**Comment by sipickles** [comment]
Now fixes all tests :)

**Comment by sipickles** [comment]
Ready for review

File: test/sync/test_sync.py [hunk 1]
```diff
@@ -19,57 +19,52 @@ def setUpClass(cls):
         cls.client = Client(environment.ATHERA_API_TEST_REGION, cls.token)
 
     def test_get_mounts(self):
-        """ Test we can get groups for the authenticated user.
-        Will only work with Python 2. Python 3 will segfault - TODO check this still applies
-        """
+        """ Test we can get groups for the authenticated user. """
```
Comment (ulzar) [inline test/sync/test_sync.py]: *Test we can get mounts

File: test/sync/test_sync.py [hunk 2]
```diff
@@ -19,57 +19,52 @@ def setUpClass(cls):
...
         for _, err in filesGenerator:
             self.assertIsNone(err, "Got unexpected error: {}".format(err))
 
-           
-
     def test_download(self):
-        """ Test we can get files for the authenticated user.
-        Will only work with Python 2. Python 3 will segfault - TODO check this still applies
-        """
+        """ Test we can get files for the authenticated user. """
```
Comment (ulzar) [inline test/sync/test_sync.py]: Test we can download files

**Pull request merged**